core-bundle/src/Twig/Extension/ImageExtension.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Twig\Extension;
+
+use Contao\CoreBundle\Twig\Runtime\FigureRendererRuntime;
+use Contao\CoreBundle\Twig\Runtime\PictureConfigurationRuntime;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class ImageExtension extends AbstractExtension
+{
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction(
+                'contao_figure',
+                [FigureRendererRuntime::class, 'render'],
+                ['is_safe' => ['html']]
+            ),
+            new TwigFunction(
+                'picture_configuration',
```

Comment:
Maybe use `picture_config` instead? Would be shorter. (Your choice though.)

Reply:
I'm fine with both. I like the shorter name, but the longer one is consistent with the class name. :thinking: 

/cc @ausi

Reply:
+1 for `picture_config`

Reply:
Shortened in c287ab880953bb836a0cc653f64da57ae9b41a60